### PR TITLE
Purify es6

### PR DESCRIPTION
**Pull request by Ffloriel**
[Build Status](https://travis-ci.org/Ffloriel/purifycss)
[dependencies Status](https://david-dm.org/ffloriel/purifycss)

- ES6 version
- Using Rollup to build es and commonjs module
- Using Babel to transpile to es5
- Using jest for testing
- Correct the README
- Update dependencies
- Add cleancss options

### CODE REVIEW

**Comment by Illyism** [comment]
Thanks for all the work! It looks great. There is an issue with the command line tool. You need to include the `lib/` folder as well, otherwise it is not transferred to the installation directory.

We would need to check if this change does not have any unforeseen breaking changes.

**Comment by Ffloriel** [comment]
Thanks!
I removed files from package.json. If we need to remove files, we can create a .npmignore.

File: src/utils/FileUtil.js [hunk 1]
```diff
@@ -1,82 +1,58 @@
-var fs = require('fs');
-var glob = require('glob');
-var UglifyJS = require('uglifyjs');
+import UglifyJS from "uglifyjs"
```
Comment (phil-lgr) [inline src/utils/FileUtil.js]: here make sense to use uglify-js ? instead of uglifyjs which seems to be deprecated

Comment (Ffloriel) [inline src/utils/FileUtil.js]: Thanks! Make sense indeed.

**Comment by Illyism** [comment]
Looks great. I've merged this. Let's try to clear a few more issues and push for an update.

**Pull request merged**